debian/bionic/foreman/control

```diff
@@ -239,3 +239,14 @@ Description: metapackage providing journald dependencies for Foreman
  a flexible systems management web application.
  .
  The foreman package should be installed along with this package.
+
+Package: foreman-service
+Architecture: all
+Section: web
+Priority: extra
+Depends: foreman (= ${binary:Version}), ruby-puma, libsystemd-dev, ${misc:Depends}
+Description: package providing Foreman services running via puma
+ This package provides Foreman systemd service for Foreman,
+ a flexible systems management web application.
+ .
+ The foreman package should be installed along with this package.
```

Comment:
you depend on it, so this seems rather given?

Reply:
Just copy pasting and keeping things in line with others.